.github/workflows/benchmark.yml

```diff
@@ -181,7 +182,7 @@ jobs:
 
     - name: Exclude very huge data
       if: steps.cache-data.outputs.cache-hit == 'true'
-      run: rm -rf data/81cd05d0 data/71455799 data/8cda00f3 data/a09d9e50
+      run: rm -rf data/8* data/7* data/a* data/2* data/0* data/f*  data/b* data/d*
```

Comment:
Isn't it remove too many directories..?
Could you let me know how many directories and files are removed, and how much the coverage is decreased?

Reply:
it is only for performance benchmark to reduce common time less than full benchmark with the scores.

Reply:
I see. I understood.